### PR TITLE
[SF-4] Register page

### PR DESCRIPTION
**Pull request by chantalchawki**

### CODE REVIEW

File: packages/web-client/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import Logo from '../../assets/icons/skillfuze-logo.svg';
+
+interface Props {
+  controls?: React.ReactNode;
+}
+
+const Navbar: React.FC<Props> = (props: Props) => (
+  <nav className="flex py-3 px-5">
+    <div className="flex-grow">
+      <Logo style={{ width: '7.5rem' }} />
```
Comment (Karimelsayed98) [inline packages/web-client/components/Navbar/Navbar.tsx]: I think we should use tailwind, I was told to use approximate values.

File: packages/web-client/pages/register/index.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [error, setError] = useState<any>({});
+
+  const onSubmit = async (event: React.FormEvent<HTMLFormElement>): Promise<void> => {
+    event.preventDefault();
+
+    try {
+      await AuthService.getInstance().register({
```
Comment (Karimelsayed98) [inline packages/web-client/pages/register/index.tsx]: Changed to AuthService.instance in login branch

Comment (khaled-hamam) [reply packages/web-client/pages/register/index.tsx]: I can't pull pull right now or we'll have the same problem regarding merging a branch = merging the other.
Will fix it later.

File: packages/web-client/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (khaled-hamam) [reply packages/web-client/components/Navbar/Navbar.tsx]: Tailwind gives us a 6rem, and 8rem values. to exactly match the size we want, a custom width had to be used.

https://tailwindcss.com/docs/width

**Pull request merged**